static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -8,12 +8,14 @@ import {DropdownMenu, MenuItemProps} from 'sentry/components/dropdownMenu';
 import IdBadge from 'sentry/components/idBadge';
 import LoadingIndicator from 'sentry/components/loadingIndicator';
 import useOnboardingDocs from 'sentry/components/onboardingWizard/useOnboardingDocs';
+// TODO: should be time to move this
+import {Flex} from 'sentry/components/profiling/flex';
```

Comment:
😬 We spent a bunch of time in the past removing a component like this https://github.com/getsentry/sentry/pulls?q=is%3Apr+reflexbox+is%3Aclosed+

Reply:
So I've learned... I brought this up in FE-TSC before https://www.notion.so/sentry/FE-TSC-Proposal-New-Layout-Primitive-Components-06ef641865de4f0eabcb60f5f340573f?pvs=4

i can't bring myself to keep writing styled components everytime i need some minor layout changes.

if we have really strong opinions about it, i can stop using it. realistically i don't really want a raw `Flex` component, i think we need something closer to a `Stack` and `Inline` component that has sane defaults

Reply:
I agree we need higher level primitives like a `Stack` or `Inline` - I think in the mean time we just stick to styled components for now.